### PR TITLE
Added web development resources

### PR DESCRIPTION
**Pull request by Virentanti**

### CODE REVIEW

**Comment by Virentanti** [comment]
Contribution #6

File: README.md [hunk 1]
```diff
@@ -48,3 +48,9 @@ A collection of helpful and free dev resources.
 - [Dribbble](https://dribbble.com)
 - [SiteInspire](https://www.siteinspire.com)
 - [WebDesign Inspiration](https://www.webdesign-inspiration.com)
+
+### Web devlopment resources
```
Comment (nataliepina) [inline README.md]:
```suggestion
### Learning Resources
```
There was a typo to fix here but I think Learning Resources better defines the section.

File: README.md [hunk 2]
```diff
@@ -48,3 +48,9 @@ A collection of helpful and free dev resources.
...
+- [Scrimba](https://scrimba.com)
+- [Edyoda](https://www.edyoda.com/)
```
Comment (nataliepina) [inline README.md]: Can you remove Edyoda or correct it this one doesn't work for me.

Comment (Virentanti) [reply README.md]: Made the relevant changes

**Comment by nataliepina** [comment]
Looks good, thanks, merged it in!

**Pull request merged**